modules/ROOT/pages/dw-core-annotations.adoc

```diff
@@ -29,7 +28,6 @@ especially with complex Object types.
 | `@Lazy()`
 | Annotation that marks a Variable declaration to be Lazily initialized.
```

Comment:
MDA: RE @Lazy(), we should add that's it's for internal purposes also, like we have here:  https://docs.mulesoft.com/mule-runtime/4.3/dataweave-variables#lazy-annotation Perhaps same description? 

@Lazy() Annotation
When you are writing a DataWeave script in Studio, you might see the internal annotation @Lazy() in autocompletion suggestions. This annotation is not intended for general use and does not improve the performance of your scripts.

Reply:
TODO: Needs copyedit in src.